s3/pkg/datastore/huawei/huawei.go

```diff
@@ -38,16 +38,31 @@ type OBSAdapter struct {
 }
 
 func (ad *OBSAdapter) BucketDelete(ctx context.Context, in *pb.Bucket) error {
+	_, err := ad.client.DeleteBucket(in.Name)
+	if err != nil {
+		log.Errorf("bucket deletion is failed :\n", err)
```

Comment:
nit: please remove redundant ```\n``` from added code

Reply:
removed

Reply:
still ```\n``` in log statement

Reply:
still can see ```\n``` in log statement :)